src/numJewelsInStones.js

```diff
@@ -28,7 +28,16 @@
  * @return {number}
  */
 function numJewelsInStones(jewels, stones) {
-  // write code here
+  let stonesSum = 0;
+  const normalizedStones = stones.replace(/[^a-zA-Z]/g, '');
```

Comment:
если мы хотим избавиться от ненужных символов, второй аргумент опционален
```suggestion
  const normalizedStones = stones.replace(/[^a-zA-Z]/g);
```